ext/scheduler/airflow2/resources/base_dag.py

```diff
@@ -227,16 +227,15 @@
 {{- end}}
 
 {{- range $_, $dependency := $.Job.ExternalDependencies.OptimusDependencies}}
-{{ $identity := print $dependency.Name "-" $dependency.ProjectName "-" $dependency.JobName }}
-wait_{{$identity | replace "-" "__dash__" | replace "." "__dot__"}} = SuperExternalTaskSensor(
+wait_{{$dependency.JobName | replace "-" "__dash__" | replace "." "__dot__"}} = SuperExternalTaskSensor(
```

Comment:
i believe we can do this change only on the id's the name of the sensor can still be the same.

Reply:
we can set the name too such that it is evident to users that it is external dependency